src/page/page_cache.rs

```diff
@@ -235,7 +282,9 @@ impl<M> PageCache<LockFreeLog, M>
         partial_pages.reverse();
         let partial_pages = partial_pages;
 
-        let mut head = head;
+        let to_evict = self.lru.accessed(pid, std::mem::size_of_val(&partial_pages));
+        self.page_out(to_evict);
```

Comment:
the lru may tell us at any time that we need to clear-up some space, then we use `page_out` to clean up memory.